src/util/fs.js

```diff
@@ -808,3 +821,23 @@ export async function makeTempDir(prefix?: string): Promise<string> {
   await mkdirp(dir);
   return dir;
 }
+
+export function expandPath(path: string): string {
```

Comment:
Is it an fs method?
From node perspective it would rather be in `path` modules rather than `fs`.